server/app/serializers/fusor/deployment_serializer.rb

```diff
@@ -49,23 +50,22 @@ class DeploymentSerializer < ActiveModel::Serializer
                :cloudforms_db_disk_size,
                :created_at, :updated_at
 
-
     has_one :organization, serializer: ::OrganizationSerializer
-    has_one :lifecycle_environment, serializer: ::LifecycleEnvironmentSerializer
+    has_one :lifecycle_environment, serializer: LifecycleEnvironmentSerializer
     # has one engine
-    has_one :discovered_host, serializer: ::HostBaseSerializer
+    has_one :discovered_host, serializer: DiscoveredHostSerializer
     # has many hypervisors
-    has_many :discovered_hosts, serializer: ::HostBaseSerializer
+    has_many :discovered_hosts, serializer: DiscoveredHostSerializer
```

Comment:
We should consider changing discovered_host -> rhev_engine_host and discovered_hosts -> rhev_hypervisor_hosts. Our model has changed since we've added OpenShift and having the front end refer to discovered hosts when the server model is something different is confusing and I've had to explain it to several people when they joined our team.